Modules/store.js

```diff
@@ -0,0 +1,29 @@
+export default class Store {
+  static getbooks = () => {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = this.getbooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(title) {
```

Comment:
- [ ] Please use `arrow` syntax for functions. ( 👉 ES6 syntax, Project requirements).